design-tokens/tokens/$themes.json

```diff
@@ -6,89 +6,90 @@
     "selectedTokenSets": {
       "layer1/colors": "source",
       "layer2/light": "enabled",
-      "layer3/base": "enabled",
-      "layer3/light": "enabled"
+      "layer3/light": "enabled",
+      "layer1/base": "source"
     },
```

Comment:
I think this is the issue: we aren't using the layer3 base?

Reply:
oh, maybe this PR should be rebased on our common branch?

Reply:
I enabled layer 3 - base on the base theme. Can you check if this is fixed now @jeremy-davis-sonarsource ? Thanks